pkg/core/blockchain.go

```diff
@@ -934,6 +947,11 @@ func (bc *Blockchain) GetGoverningTokenBalance(acc util.Uint160) (*big.Int, uint
 	return &neo.Balance, neo.LastUpdatedBlock
 }
 
+// GetDepositFor returns GAS amount deposited to Notary contract for the specified account.
+func (bc *Blockchain) GetDepositFor(acc util.Uint160) *big.Int {
+	return bc.contracts.Notary.BalanceOf(bc.dao, acc)
```

Comment:
I see how dependency on native contracts may prevent this from being done outside of `core`. May be we can do something like what was done in #1427.
E.g. updating `GetBalance()` function during `OnPersistEnd()`.

Reply:
We have `(*NotaryPool).PostBlock` now, which can make use of `Blockchainer`. Do we still need this updating function?